src/components/NavBar/AccountMenu.tsx

```diff
@@ -61,8 +78,14 @@ const AccountMenu = () => {
           vertical: "bottom",
           horizontal: "center",
         }}
+        sx={{ ".MuiPopover-paper": styles.popover }}
```

Comment:
If we move the selector to the styles object I think we can keep the syntax consistent with the rest of the app

```
const styles = {
  popover: {
    ".MuiPopover-paper": {
      padding: 1,
      borderRadius: 3,
    },
  },
};

...

<Popover>
  ...
  sx={styles.popover}
</Popover>
```

Reply:
Done